satpy/readers/fci_l1c_nc.py

```diff
@@ -182,6 +182,18 @@ def _get_channel_name_from_dsname(dsname):
 
     return channel_name
 
+# Platform names according to the MTG FCI L1 Product User Guide,
+# EUM/MTG/USR/13/719113 from 2019-06-27, pages 32 and 124, are MTI1, MTI2,
+# MTI3, and MTI4, but we want to use names such as described in WMO OSCAR
+# MTG-I1, MTG-I2, MTG-I3, and MTG-I4.
+#
+# Not sure how the numbering will be considering MTG-S1 and MTG-S2 will be launched
+# in-between.
+_platform_name_translate = {
+    "MTI1": "Meteosat-12",
+    "MTI2": "MTG-I2",
+    "MTI3": "MTG-I3",
+    "MTI4": "MTG-I4"}
```

Comment:
I would move this to `fci_base.py` so it can also be used by `fci_l2_nc.py` (see #3026).  Or even an `mtg_base.py` to be used by both FCI and LI.

Reply:
Good idea. Wasn't aware of that otherwise would have done it directly :-).